package.json

```diff
@@ -18,9 +18,9 @@
     "test": "npx ng test",
     "lint": "npx ng lint",
     "e2e": "npx ng e2e",
-    "test-dev": "npx cypress run -c baseUrl=https://dev.dockstore.net -s \"cypress/integration/manualTests/monitor.ts,cypress/integration/manualTests/basic-enduser.ts\"",
-    "test-staging": "npx cypress run -c baseUrl=https://staging.dockstore.org -s \"cypress/integration/manualTests/monitor.ts,cypress/integration/manualTests/basic-enduser.ts\"",
-    "test-prod": "npx cypress run -c baseUrl=https://dockstore.org -s \"cypress/integration/manualTests/monitor.ts,cypress/integration/manualTests/basic-enduser.ts\""
+    "test-dev": "npx cypress run -c baseUrl=https://dev.dockstore.net -s \"cypress/integration/manualTests/monitor.ts,cypress/integration/manualTests/basic-enduser.ts,cypress/integration/manualTests/auth-tests.ts\"",
```

Comment:
I'm thinking it might be good to have the auth and non-auth smoke tests separate. When we deploy 1.9 to staging or prod, it would be good to be able to manually invoke the smoke tests, and if the auth user is not set up in a particular environment...

Reply:
added separate test-no-auth and test-auth scripts